app/controllers/steps_controller.rb

```diff
@@ -0,0 +1,27 @@
+class StepsController < ApplicationController
+  include Wicked::Wizard::Translated
+
+  before_action :authenticate_user!
+
+  steps(*User.steps)
+
+  def show
+    render_wizard
+  end
+
+  def update
+    wizard_value = wizard_value(step)
```

Comment:
Get the reverse translation from the URL to step (`/3` -> `stage_3`) and store it in a variable.